python/lsst/dax/apdb/apdb.py

```diff
@@ -155,13 +156,26 @@ def from_uri(cls, uri: ResourcePathExpression) -> Apdb:
         Parameters
         ----------
         uri : `~lsst.resources.ResourcePathExpression`
-            URI pointing to a file with serialized configuration.
+            URI pointing to a file with serialized configuration. IF ``uri`` is
+            a string with "label:" prefix then the label name which follows the
+            prefix will be searched in APDB index file whose location is
+            determined by ``DAX_APDB_INDEX_URI`` environment variable.
```

Comment:
This wording requires the reader to already know what an index file is _for_. Perhaps:
```suggestion
            URI pointing to a file with serialized configuration, or a string 
            with a "label:" prefix. In the latter case, the configuration will
            be looked up from an APDB index file using the label name that
            follows the prefix. The APDB index file's location is determined by
            the ``DAX_APDB_INDEX_URI`` environment variable.
```